server/recceiver/cfstore.py

```diff
@@ -86,17 +121,21 @@ def __commit__(self, TR):
         for pv in delrec:
             if iocid in self.channel_dict[pv]:
                 self.channel_dict[pv].remove(iocid)
-                self.iocs[iocid]["channelcount"] -= 1
+                if iocid in self.iocs:
+                    self.iocs[iocid]["channelcount"] -= 1
```

Comment:
I am not completely comfortable with this either. I did have an issue when doing my tests (restarting the same ioc on the same machine multiple times) which resulted in this check...
The main goal of my changes is to minimize the network calls and to allow the parsing of info tags in the iocs